enterprise.constants

```diff
@@ -9,6 +9,10 @@ KIT_ID='adsk'
 # Define your GitHub Enterprise server
 GITHUB_SERVER='git.yourcompany.com'
 
+
+# Add the protocol used to access GitHub server: http or https
+GITHUB_PROTOCOL='https'
```

Comment:
I think we should follow your naming convention from `GHE_USER` etc. here and call this `GHE_HTTP`?